src/Views/Components/ListTable/FormSelect.tsx

```diff
@@ -0,0 +1,43 @@
+import styles from "@givewp/components/ListTable/ListTablePage.module.scss";
+
+//TODO: extract SearchableSelect component from FormSelect
+export const FormSelect = ({options, name, placeholder = '', ariaLabel = '', onChange}) => {
+    return (
+        <>
+            <input
+                type='search'
+                className={styles.searchInput}
+                list={`giveSearchSelect-${name}`}
+                onChange={updateSearchableSelect(options, name, onChange)}
+                autoComplete={'off'}
+                aria-label={ariaLabel}
+                placeholder={placeholder}
+            />
+            <datalist
+                className={styles.statusFilter}
+                id={`giveSearchSelect-${name}`}
+                onChange={onChange}
+            >
+                {options.map(({value, text}) => (
+                    <option key={`${value}${text}`} value={text}>
+                        {value === '0' ? '' : `ID ${value}`}
```

Comment:
@LMNTL did we leave the `ID` text intentionally not translatable?

Reply:
@LMNTL No, that was a mistake on my part. I will push a fix for that to the scope branch.